liblangutil/EVMVersion.h

```diff
@@ -85,6 +87,8 @@ class EVMVersion:
 
 	static std::optional<EVMVersion> fromString(std::string const& _version)
 	{
+		if (_version == "current")
+			return current();
```

Comment:
Note that this exposes the new alias everywhere, including the `--evm-version` option on the CLI. That wasn't really my intention here. I was thinking about just making it available in the `EVMVersion` setting in tests as a special value.

I would not be against having such an alias. Even internally the fact that `EVMVersion{}` means automatically the latest version may not be obvious (it wasn't to me at first), even if it's convenient. But it's a bigger change we should discuss more broadly first. It would also need a changelog entry and should be documented.

Reply:
It was also anything but obvious to me but you are right of course, this is a change that is outside-facing. Let's discuss it next week perhaps?

Reply:
Sure. In this PR I'd still limit it to tests though so that we can merge it.

Reply:
Done